src/spell.sol

```diff
@@ -1,101 +1,229 @@
+       root.relyContract(COORDINATOR_OLD, self);
+       root.relyContract(RESERVE, self);
+       root.relyContract(MGR, self);
+       root.relyContract(FEED, self);
+       
+       migrateClerk();
+       migrateCoordinator();
+       migratePoolAdmin();
+     }  
+
+     function migrateCoordinator() internal {
+        // migrate state
+        MigrationLike(COORDINATOR).migrate(COORDINATOR_OLD);
+
+        // migrate dependencies
+        DependLike(COORDINATOR).depend("assessor", ASSESSOR);
+        DependLike(COORDINATOR).depend("juniorTranche", JUNIOR_TRANCHE);
+        DependLike(COORDINATOR).depend("seniorTranche", SENIOR_TRANCHE);
+
+        DependLike(CLERK).depend("coordinator", COORDINATOR);
+
+        DependLike(SENIOR_TRANCHE).depend("coordinator", COORDINATOR);
+        DependLike(JUNIOR_TRANCHE).depend("coordinator", COORDINATOR);
+
+        // migrate permissions
+        AuthLike(ASSESSOR).rely(COORDINATOR); 
+        AuthLike(ASSESSOR).deny(COORDINATOR_OLD);
+        AuthLike(JUNIOR_TRANCHE).rely(COORDINATOR); 
+        AuthLike(JUNIOR_TRANCHE).deny(COORDINATOR_OLD); 
+        AuthLike(SENIOR_TRANCHE).rely(COORDINATOR);
+        AuthLike(SENIOR_TRANCHE).deny(COORDINATOR_OLD);
+     }
+
+     function migratePoolAdmin() internal {
+        // setup dependencies 
+        DependLike(POOL_ADMIN).depend("assessor", ASSESSOR);
+        // DependLike(POOL_ADMIN).depend("lending", CLERK); // set in clerk migration
+        DependLike(POOL_ADMIN).depend("seniorMemberlist", SENIOR_MEMBERLIST);
+        DependLike(POOL_ADMIN).depend("juniorMemberlist", JUNIOR_MEMBERLIST);
+        DependLike(POOL_ADMIN).depend("navFeed", FEED);
+        DependLike(POOL_ADMIN).depend("coordinator", COORDINATOR);
+
+        // setup permissions
+        AuthLike(ASSESSOR).rely(POOL_ADMIN);
+        AuthLike(ASSESSOR).deny(POOL_ADMIN_OLD);
+        // AuthLike(CLERK).rely(POOL_ADMIN); // set in clerk migration
+        // AuthLike(CLERK).deny(POOL_ADMIN_OLD);
+        AuthLike(JUNIOR_MEMBERLIST).rely(POOL_ADMIN);
+        AuthLike(JUNIOR_MEMBERLIST).deny(POOL_ADMIN_OLD);
+        AuthLike(SENIOR_MEMBERLIST).rely(POOL_ADMIN);
+        AuthLike(SENIOR_MEMBERLIST).deny(POOL_ADMIN_OLD);
+        AuthLike(FEED).rely(POOL_ADMIN);
+        AuthLike(FEED).deny(POOL_ADMIN_OLD);
+        AuthLike(COORDINATOR).rely(POOL_ADMIN);
+        AuthLike(COORDINATOR).deny(POOL_ADMIN_OLD);
+
+        // set lvl3 admins
+        AuthLike(POOL_ADMIN).rely(LEVEL3_ADMIN1);
+        // set lvl1 admins
+        PoolAdminLike(POOL_ADMIN).setAdminLevel(LEVEL1_ADMIN1, 1);
+        PoolAdminLike(POOL_ADMIN).setAdminLevel(LEVEL1_ADMIN2, 1);
+        PoolAdminLike(POOL_ADMIN).setAdminLevel(LEVEL1_ADMIN3, 1);
+        PoolAdminLike(POOL_ADMIN).setAdminLevel(LEVEL1_ADMIN4, 1);
+        PoolAdminLike(POOL_ADMIN).setAdminLevel(LEVEL1_ADMIN5, 1);
+        PoolAdminLike(POOL_ADMIN).setAdminLevel(AO_POOL_ADMIN, 1);
+        PoolAdminLike(POOL_ADMIN).setAdminLevel(MEMBER_ADMIN, 1);
```

Comment:
The member admin needs to be relied on the memberlists directly, not on the pool admin. See: https://github.com/centrifuge/tinlake/blob/develop/src/lender/deployer.sol#L236